src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts

```diff
@@ -154,4 +154,41 @@ describe('ProjectListComponent', () => {
     component.switchStatus(itemData);
     expect(component.showModal).toBeFalse();
   });
+
+  it('getProjectType should be called to display it in projects table', () => {
+    const nameType = {
+      id: '1234',
+      name: 'BK',
+      description: 'test',
+    };
+    const id = '1234';
+    component.projectsTypes = [nameType];
+    component.ngOnInit();
+    const nameTest = component.getProjectType(id);
+    expect(nameTest).toBe('BK');
```

Comment:
The **nameTest** get the id of the project.
and in the expect should compare the **id**
`expect(idTest).toBe('1223');`

and you can create another expect for  **name**
`NAME: expect(nameTest).toBe('BK');`

Reply:
The nameTest gets the name of the projectType to print it in the table, that's why I changed the `getPProjectType`  function to  `getProjectTypeName` in the next commit.